packs/github/config.schema.yaml

```diff
@@ -0,0 +1,52 @@
+---
+token:
+  description: "GitHub oAuth Token"
+  type: "string"
+  default: "{{user.github_enterprise_oauth_token}}"
+  required: true
+
+user: 
+  description: "GitHub Username"
+  type: "string"
+
+password:
+  description: "GitHub Password"
+  type: "string"
+
+default:
```

Comment:
And it looks like in another place `github_type` parameter name is used so it's also good to be consistent and use the same name across the board :)